### ISSUE TITLE
"Sorry, but you are not permitted to view this content." with parental controls turned off

### ISSUE THREAD

**Issue by joaosilvajci** [HookedBehemoth/ComicNX#10]
FW: 7.0.1

**Comment by HookedBehemoth** [comment]
I believe I never tested on a lower firmware than 8.0.0.
I'll rewrite this after the next screen-nx release.

**Comment by HookedBehemoth** [comment]
Do you use ulaunch by any chance?

**Comment by joaosilvajci** [comment]
No, I don't. It's just the normal .nro drop in the folder and opening via the default cfw launcher.
In case it's relevant, the parental controls were active when i first opened but withouth age restriction. I then disabled, tried again and the same issue.

**Comment by HookedBehemoth** [comment]
Do you have a folder /atmosphere/contents/0100000000001001?
